src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -294,6 +294,23 @@ public void scheduleEmailsForSending(List<EmailWrapper> emails) {
         }
     }
     
+    /**
+     * Schedules production of searchable documents
+     * Task queue handles this so that instructor account creation depends on less database calls
```

Comment:
Usages of an API (e.g. called by abcd) should not be part of documentation. If this is necessary for future developers, comment on the call site, e.g. "task queue is used here to reduce calls to database that can otherwise cause this action to take too long to finish".